llvm-external-projects/iree-dialects/BUILD

```diff
@@ -303,40 +303,14 @@ cc_library(
     ],
 )
 
-cc_library(
+alias(
```

Comment:
Can we add a TODO to fix this and leave the issue open? I see this as a workaround more than a fix.